src/main/java/frc/robot/Robot.java

```diff
@@ -48,6 +53,11 @@ public class Robot extends TimedRobot {
   public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and use the subsystems needed
     // for the specific robot
+
+    String logfolder = "/home/lvuser";
```

Comment:
To avoid filling up the filesystem we could do:

- recommended -  save logs to usb stick so they can be analyzed immediately after the game.
- periodically  ssh to robot and delete old log files. 
- during robotInit check and remove files older then a week ( it may increase start time )

Reply:
And a button on the shuffleboard